meta-lmp-bsp/recipes-security/optee/optee-os-fio-bsp.inc

```diff
@@ -121,3 +121,15 @@ EXTRA_OEMAKE:append:uz3eg-iocc-sec = " \
     CFG_REE_FS=n CFG_RPMB_FS=y CFG_RPMB_WRITE_KEY=y \
     CFG_RPMB_FS_DEV_ID=0 \
 "
+
+EXTRA_OEMAKE:append:stm32mp15-eval-sec = " \
+    CFG_STM32MP15_HUK=y \
+    CFG_STM32MP15_HUK_BSEC_KEY_0=60 \
+    CFG_STM32MP15_HUK_BSEC_KEY_1=61 \
+    CFG_STM32MP15_HUK_BSEC_KEY_2=62 \
+    CFG_STM32MP15_HUK_BSEC_KEY_3=63 \
+    CFG_REE_FS=n CFG_RPMB_FS=y \
+    CFG_STM32_BSEC_WRITE=y \
+    CFG_RPMB_WRITE_KEY=y \
```

Comment:
I don't think we normally enable `CFG_RPMB_WRITE_KEY` for standard LmP builds.  This should only be enabled for the mfgtools.
In the case of `uz3eg-iocc-sec` above, there are no mfgtool builds, so we enable it there.
@ricardosalveti can confirm.